test/javascripts/acceptance/chat-test.js

```diff
@@ -725,6 +725,20 @@ acceptance("Discourse Chat - without unread", function (needs) {
     );
   });
 
+  test("message selection and live pane buttons for regular staff", async function (assert) {
```

Comment:
I'm not sure which is wrong here because the description says for staff but `admin` and `moderator` is `false` below.

Reply:
Ah brain fart, should have said `for regular user`